designer/server/src/main/scala/pl/touk/nussknacker/ui/services/ScenarioAttachmentService.scala

```diff
@@ -0,0 +1,61 @@
+package pl.touk.nussknacker.ui.services
+
+import com.typesafe.scalalogging.LazyLogging
+import org.apache.commons.io.IOUtils
+import org.apache.commons.io.input.BoundedInputStream
+import pl.touk.nussknacker.engine.api.process.{ProcessId, VersionId}
+import pl.touk.nussknacker.ui.config.AttachmentsConfig
+import pl.touk.nussknacker.ui.process.repository.ProcessActivityRepository
+import pl.touk.nussknacker.ui.security.api.LoggedUser
+import pl.touk.nussknacker.ui.services.ScenarioAttachmentService.{AttachmentDataWithName, AttachmentToAdd}
+
+import java.io.InputStream
+import scala.concurrent.{ExecutionContext, Future}
+import scala.util.Using
+
+class ScenarioAttachmentService(config: AttachmentsConfig, scenarioActivityRepository: ProcessActivityRepository)
+    extends LazyLogging {
+
+  def saveAttachment(
+      scenarioId: ProcessId,
+      scenarioVersionId: VersionId,
+      originalFileName: String,
+      inputStream: InputStream
+  )(implicit ec: ExecutionContext, loggedUser: LoggedUser): Future[Unit] = {
```

Comment:
maybe ExecutionContext in the constructor is enough? Or maybe we really need it at the method level?

Reply:
Done